geomstats/geometry/poincare_half_space.py

```diff
@@ -36,23 +36,26 @@ def __init__(self, dim, scale=1):
         self.metric = PoincareHalfSpaceMetric(self.dim, self.scale)
 
     def belongs(self, point):
-        """Test if a point belongs to the hyperbolic space.
+        """Evaluate if a point belongs to the manifold of normal distributions.
 
-        Test if a point belongs to the hyperbolic space based on
-        the poincare ball representation.
+        The statistical manifold of normal distributions is the upper
+        half plane.
 
         Parameters
         ----------
-        point : array-like, shape=[..., dim]
-            Point to be tested.
+        point : array-like, shape=[..., 2]
```

Comment:
Why this restriction in the docstring to dimension 2 ? The code is valid for any dimension.